package.json

```diff
@@ -15,7 +15,8 @@
       "android:start": "cd examples/end-user-happ && nix develop .#androidDev --command bash -c \"pnpm run tauri android dev\"",
       "publish:holochain-service-client": "cd libraries/holochain-service-client && ./gradlew publish",
       "publish:local:holochain-service-client": "cd libraries/holochain-service-client && ./gradlew publishToMavenLocal",
-      "test:holochain-service-client": "cd libraries/holochain-service-client && ./gradlew connectedDebugAndroidTest"
+      "test:holochain-service-client": "cd libraries/holochain-service-client && ./gradlew connectedDebugAndroidTest",
+      "test:tauri-plugin-holochain-service": "cd crates/tauri-plugin-holochain-service/android && ./gradlew connectedDebugAndroidTest"
```

Comment:
I will add these to CI in a follow-up PR. I think it will be cleaner to pull the holochain service into a separate kotlin library first.